website/content/docs/concepts/storage.mdx

```diff
@@ -0,0 +1,110 @@
+---
+layout: docs
+page_title: Storage
+sidebar_title: Storage
+description: >-
+  Vault relies on external storage to save its information.
+---
+
+# Storage
+
+As described on our [Architecture](/docs/internals/architecture) page, Vault's
+storage backend is untrusted storage used purely to keep encrypted information.
+
+## Supported Storage Backends
+
+For enterprise customers, HashiCorp provides official support for Consul and
+Vault's Integrated Storage as storage backends.
+
+Many other options for storage are available with community support - see our
+[Storage Configuration](/docs/configuration/storage) section for more
+information.
+
+## Backups
+
+Due to the highly flexible nature of Vault's potential storage configurations,
+providing exact guidance on backing up Vault is challenging.
+
+When backing up Vault, there are two pieces to consider:
+
+1. Vault's encrypted data in the storage backend
+2. Configuration files and management scripts for running the Vault server
+
+There's also a big question - what is the error case you're trying to guard
+against by saving a backup?
+
+### The Big Question - Why Take Backups?
+
+It's important to consider the question of "why take a backup" while developing
+your ongoing backup and disaster recovery strategy.
+
+Taking a backup is recommended prior to upgrades, as downgrading Vault storage
+is not always possible. Generally, a backup is recommended any time a major
```

Comment:
We may want to reword this to keep people from thinking that it is sometimes possible to downgrade storage. Some ideas: "downgrading Vault storage is not recommended", or "not supported"?

Reply:
Yeah, that's fair.